### PR TITLE
Add access to users to dhcuratedhive schema

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

**Comment by accorvin** [comment]
Any reason we're not putting these users into a group instead?

File: kfdefs/base/trino/trino-config-secret.yaml [hunk 1]
```diff
@@ -102,6 +103,16 @@ stringData:
           "schema": "usir",
           "owner": true
         },
+        {
+          "group": "rhods",
+          "schema": "dhcuratedhive",
+          "owner": true
+        },
+        {
+          "user": "jdemoss|egranger|cchase|gmoutier",
```
Comment (accorvin) [inline kfdefs/base/trino/trino-config-secret.yaml:112]: We no longer need this since they were added to the group, right?

**Review by anishasthana — APPROVED** [review]

**Pull request merged**